### PR TITLE
US19592 poll for donations

### PR DESCRIPTION
**Pull request by jpc11623**
Initial commit of WIP code

### CODE REVIEW

File: Crossroads.Service.Finance/Controllers/PollingController.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        public PollingController(IPushpayService pushpayService)
+        {
+            _pushpayService = pushpayService;
+        }
+
+        /// <summary>
+        ///    Sync settlements from pushpay into MP
+        /// </summary>
+        /// <remarks>
+        ///    Called via a SyncPushpaySettlements windows scheduled task at 1pm every day
```
Comment (candrews55) [inline Crossroads.Service.Finance/Controllers/PollingController.cs:25]: I don't think this comment matches what the code is doing

File: Crossroads.Service.Finance/Controllers/PollingController.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        /// </remarks>
+        [HttpPost("donations")]
+        [ProducesResponseType(200)]
+        [ProducesResponseType(500)]
+        public async Task<IActionResult> PollDonations([FromQuery] DateTime startDate, [FromQuery] DateTime endDate)
```
Comment (candrews55) [inline Crossroads.Service.Finance/Controllers/PollingController.cs:30]: Should we even have the params if we are not going to use them

File: Crossroads.Service.Finance/Controllers/PollingController.cs (hunk 1 above)
Comment (jpc11623) [reply Crossroads.Service.Finance/Controllers/PollingController.cs]: Fixed

File: Crossroads.Service.Finance/Controllers/PollingController.cs (hunk 2 above)
Comment (jpc11623) [reply Crossroads.Service.Finance/Controllers/PollingController.cs]: Fixed

**Pull request merged**